decidim-core/app/packs/src/decidim/editor/extensions/video_embed/index.js

```diff
@@ -9,15 +9,15 @@ const YOUTUBE_REGEX_GLOBAL = /^(https?:\/\/)?(www\.|music\.)?(youtube\.com|youtu
 const VIMEO_REGEX = /^(https?:\/\/)?(www\.|player\.)?(vimeo\.com)(.+)?$/;
 const VIMEO_REGEX_GLOBAL = /^(https?:\/\/)?(www\.|player\.)?(vimeo\.com)(.+)?$/g;
 
-const isValidYoutubeUrl = (url) => {
+const isValidYouTubeUrl = (url) => {
```

Comment:
I'm hoping these functions are deemed internal...

Reply:
There are only 2 occurrences of this in the entire project.